doc/rst/technotes/libraries.rst

```diff
@@ -403,6 +403,14 @@ program.  Each library file must include the chapel runtime and standard modules
 for its own functionality and when two or more libraries are linked to a program
 this leads to multiple definitions of these functions.
 
+LLVM
+----
+
+LLVM support with ``--library`` is currently a work-in-progress.  For the
+current release, it is only supported for C with a subset of the flags, and does
```

Comment:
Can we say 1.19 here, so we don't forget to update it & it turns out wrong in 1.20?